content/synthetics/uptime_check.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Uptime Check
+kind: documentation
+beta: true
+description: Simulate and monitor HTTP requests from specific locations
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+---
+
+## Overview
+
+Uptime Checks are HTTP requests (GET, POST, PUT, etc.) executed by Datadog to your web properties or application endpoints at configurable periodic intervals from multiple locations around the world. These Checks verify not only that your applications are up and responding to requests, but that any conditions you define are met, such as response time, HTTP response code, and header or body contents.
+
+## Configuration
+### Request
+
+Define the request you want to be executed by Datadog:
+
+{{< img src="synthetics/uptime_check/uptime_check_make_request.png" alt="Uptime Check make request" responsive="true" style="width:80%;">}}
+
+1. Choose the `Method` and `URL` to query. Available `Method` are:
+    * GET
+    * POST
+    * PATCH
+    * PUT
+    * DELETE
+    * OPTIONS
+
+2. Optionally, click on **Advanced options** to enrich your request with custom Request Headers, Authentication credentials, body content, or cookies.
+
+3. Name your Uptime Check.
+4. Optionally, add tags to filter your Uptime Check in the [Synthetics page][1].
+
+5. Pick-up locations to run the test from, available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+
+6. Choose a Check frequency between "1 run per minute" to "1 run per week".
+7. Finish by clicking on **Test URL** to try out the request configuration. You should see a response preview show up in the right side of your screen.
+
+### Validation
+
+When running an Uptime Check, you must define at least one Assertions that should be monitored by Datadog. An Assertion is defined by a parameter, a comparator, and a value.
+
+| Parameter     | Comparators                | Value type                         |
+| --------      | ---------                  | --------                           |
+| Status Code   | `is`, `is not`             | Integer                            |
+| Response time | `less than`                | Integer                            |
+| Headers       | `is`, `is not`, `contains` | Strings                            |
+| body          | `contains`                 | Strings ([regex are supported][4]) |
```

Comment:
`body` comparators are `is`, `is not` and `contains`. And like `Headers` the value type is more:
- for `is (not)` always a string
- for `contains` always a regex

Side note: why is `body` not capitalized?